lib/Conversion/TritonGPUToLLVM/MinMaxFOpToLLVM.cpp

```diff
@@ -0,0 +1,75 @@
+#include "mlir/Conversion/LLVMCommon/Pattern.h"
```

Comment:
nit: can that go in lib/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVM.cpp instead?

Reply:
Would it be ok to have 2 `populate` functions in the same cpp ? The reason I did this is because this conversion takes the "enableWorkaround" (I will rename) boolean, whereas all the other elementwiseop conversions don't.

Reply:
> Would it be ok to have 2 populate functions in the same cpp ?

Yes I think this is totally fine.